mysql-test/suite/innodb/t/innodb-mdev-30321.opt

```diff
@@ -0,0 +1 @@
+--innodb_sort_buffer_size=64k
```

Comment:
I don't think we need this .opt file.

Reply:
it is optional, but I added it to decrease memory footprint and to make it run always the same as we are allocating a dtuble twice the size of "innodb_sort_buffer_size" which is 1 MB by default currently.
but sure, I will remove it